<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Maciej Migdał - bioinformatics, data analysis, IT</title>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
  <script src="https://kit.fontawesome.com/c3ebd3bc8f.js" crossorigin="anonymous"></script>
  <!-- TODO <link rel="icon" type="image/png" href="favicon.png" /> -->
</head>

<body style="background: url('rect5216.png'); background-size: 10%; background-repeat: repeat-y;">

   <section class="hero is-halfheight is-success" id="bio">
     <div class="hero-head">
        <div class="navbar-menu">
          <div class="navbar-end tabs is-right"/>

            <div class="navbar-item icon-text">
              <span class="icon">
                <i class="fas fa-regular fa-envelope"></i>
              </span>
              <span>maciej.migdal[at]bioinnowacje.com</span>
            </div>
            <div class="navbar-item icon-text">
              <span class="icon">
                <i class="fas fa-brands fa-github"></i>
              </span>
              <a href="https://github.com/mcjmigdal">github.com/mcjmigdal</a>
            </div>
            <div class="navbar-item icon-text">
              <span class="icon">
                <i class="fas fa-brands fa-linkedin"></i>
              </span>
              <a href="https://www.linkedin.com/in/mcjmigdal">in/mcjmigdal</a>
            </div>
            <div class="navbar-item icon-text">
              <span class="icon">
                <i class="fas fa-brands fa-orcid"></i>
              </span>
              <a href="https://orcid.org/0000-0002-8021-7263">orcid.org</a>
            </div>

          </div>
        </div>
     </div>
     </div>
     <div class="hero-body">
       <div class="container">
         <p class="has-text-centered">Hi I am <span class="title is-large">Maciej Migdał</span>,</p>
         </br>
         <p class="has-text-justified">
            a bioinformatician and a programmer, working on problems
            of various difficulties, be it file processing automation or gene
            expression modeling. I specialize in processing and analyzing
            Next-Generation Sequencing (NGS) data, such as RNA-seq. My expertise
            extends to Linux server administration, web application development,
            Excel automation, Raspberry Pi projects, and more!
         </p>
         </br>
         <p>
            If you face a challenge which requires a blend of technical expertise and biological
            insight, or you need an assistance in IT-related areas, do not hesitate to contact me.
         </p>

       </div>
   </section>

   <div class="container"
     >
       <section id="projects" class="section">
         <h1 class="title">Open source projects</h1>

         <div class="tile is-ancestor">
           <div class="tile is-parent is-vertical">

           <article id="xcore" class="tile is-child box is-clickable"
               onclick="window.open('https://bkaczkowski.github.io/xcore/', '_blank')">
             <p class="title">xcore</p>
             <p class="subtitle">an R package for inference of gene expression regulators</p>
             <div class="content has-text-justified"
               <p class="has-text-justified">
                 xcore is an R package for transcription factor activity modeling
                 based on known molecular signatures and user's gene expression
                 data. Accompanying <em>xcoredata</em> package provides a collection of
                 molecular signatures, constructed from publicly available
                 ChiP-seq experiments.
               </p>
             </div>
             <figure class="image">
               <img src="https://bkaczkowski.github.io/xcore/articles/flowchart.png"
                    alt="xcore workflow"
               >
             </figure>
           </article>

           <article id="sumsamstats" class="tile is-child box is-clickable"
               onclick="window.open('https://github.com/mcjmigdal/sumsamstats', '_blank')">
             <p class="title">sumsamstats</p>
             <div class="content has-text-justified"
               <p class="has-text-justified">
                 Parser and tools for summarising outputs of samtools stats, with ATAC-seq in mind.
                 Shipped together with a Shiny application to help visualize the outputs.
               </p>
             </div>
           </article>

           <article id="RNA-seq_data_analysis_course_for_biologists_may_2018" class="tile is-child box is-clickable"
               onclick="window.open('https://github.com/mcjmigdal/RNA-seq_data_analysis_course_for_biologists_may_2018', '_blank')">
             <p class="title">RNA-seq data analysis for biologists</p>
             <div class="content has-text-justified"
               <p class="has-text-justified">
                 RNA-seq workshop covering one of the simplest data analysis workflows,
                 starting with data download and finishing at differential expression calling.
               </p>
             </div>
           </article>

           <article id="wheatherstation" class="tile is-child box is-clickable"
               onclick="window.open('https://github.com/mcjmigdal/weather-station-esp32', '_blank')">
             <p class="title">Wheather station</p>
             <div class="content has-text-justified"
               <p class="has-text-justified">
                 Mini-project weather station build around BME280 sensor using ESP32
                 and E-Ink display.
               </p>
             </div>
             <figure class="image is-flex is-justify-content-center">
                 <img src="https://raw.githubusercontent.com/mcjmigdal/weather-station-esp32/master/station.png"
                      style="width: 50%; height: auto" alt=""
                 >
             </figure>
           </article>

           </div>

           <div class="tile is-parent is-vertical">

           <article id="midasHLA" class="tile is-child box is-clickable"
               onclick="window.open('https://genentech.github.io/MiDAS', '_blank')">
             <p class="title">midasHLA</p>
             <p class="subtitle">Meaningful Immunogenetic Data at Scale</p>
             <div class="content has-text-justified"
               <p>
                 midasHLA is an R package for statistical analysis of human
                 immunogenetic variation in the form of HLA and KIR types.
                 The package enables association analysis and using immunogenetic
                 data transformation functions for HLA amino acid fine mapping,
                 analysis of HLA evolutionary divergence as well as HLA-KIR
                 interactions. midasHLA closes the gap between inference of immunogenetic
                 variation and its efficient utilization to make meaningful discoveries.
               </p>
             </div>
             <figure class="image is-flex is-justify-content-center">
                 <img src="https://github.com/Genentech/midasHLA/raw/master/vignettes/MiDAS_logo.png"
                      style="width: 50%; height: auto" alt="midasHLA logo"
                 >
             </figure>
           </article>

           <article id="atacseq_pipeline" class="tile is-child box is-clickable"
               onclick="window.open('https://gitlab.com/zdglab/atacseq_pipeline', '_blank')">
             <p class="title">ATAC-seq pipeline</p>
             <div class="content has-text-justified"
               <p>
                 Data analysis pipeline implemented in Nextflow. It starts with the raw FASTQ files
                 and outputs peaks denoting nucleosome free regions toghether with count quantification.
                 The pipeline supports peak calling using information from multiple replicates.
               </p>
             </div>
           </article>

           <article id="shiitake" class="tile is-child box is-clickable"
               onclick="window.open('https://github.com/mcjmigdal/shiitake', '_blank')">
             <p class="title">shiitake</p>
             <div class="content has-text-justified"
               <p>
                 Mini-project, automatizing Shiitake mushrooms growth monitoring
                 using Raspberry Pi and Raspberry Pi Camera.
               </p>
             </div>
             <figure class="image is-flex is-justify-content-center">
                 <img src="https://raw.githubusercontent.com/mcjmigdal/shiitake/main/data/img/2021-08-09_1407.jpg"
                      style="width: 50%; height: auto;" alt="picture of shiitake mushrooms"
                 >
             </figure>
           </article>

           <article id="js_fasta_validator" class="tile is-child box is-clickable"
               onclick="window.open('https://gist.github.com/mcjmigdal/56407315d30a2252325517d0a278ae2c', '_blank')">
             <p class="title"></p>
             <div class="content has-text-justified"
               <p>
                 JavaScript FASTA format validator
               </p>
             </div>
           </article>

           </div>
         </div>
       </section>

       <section id="publications" class="section content">
         <h1 class="title">Publications</h1>
         <ul>
             <ol><b>Migdał, M.</b>, Arakawa, T., Takizawa, S., Furuno, M., Suzuki, H., Arner, E., Winata, C.L., Kaczkowski, B. <u>xcore: an R package for inference of gene expression regulators</u>. BMC Bioinformatics. 2023. 24:14. doi: https://doi.org/10.1186/s12859-022-05084-0</ol>
           <ol>Uszczynska-Ratajczak, B., Sugunan, S., Kwiatkowska, M., <b>Migdal, M.</b>, Carbonell-Sala, S., Sokol, A., Winata, C.L., Chacinska, A. <u>Profiling subcellular localization of nuclear-encoded mitochondrial gene products in zebrafish</u>. Life Sci. Alliance. 2022. 6:e202201514. doi: https://doi.org/10.26508/lsa.202201514</ol>
           <ol><b>Migdał, M.</b>, Tralle, E., Abu Nahia, K., Bugajski, Ł., Kędzierska, K.Z., Garbicz, F., Piwocka, K., Winata, C.L., Pawlak, M. <u>Multi-omics analyses of early liver injury reveals cell-type-specific transcriptional and epigenomic shift</u>. BMC Genomics. 2021. 22:904. doi: https://doi.org/10.1186/s12864-021-08173-1</ol>
           <ol>Abu Nahia, K., <b>Migdał, M.</b>, Quinn, T.A., Poon, K., Łapiński, M., Sulej, A., Liu, J., Mondal, S.S., Pawlak, M., Bugajski, Ł., Piwocka, K., Brand, T., Kohl, P., Korzh, V., Winata, C. <u>Genomic and physiological analyses of the zebrafish atrioventricular canal reveal molecular building blocks of the secondary pacemaker region</u>. Cell. Mol. Life Sci.. 2021. 78:6669-6687. doi: https://doi.org/10.1007/s00018-021-03939-y</ol>
           <ol><b>Migdal, M.</b>, Ruan, D.F., Forrest, W.F., Horowitz, A., Hammer, C. <u>MiDAS—Meaningful Immunogenetic Data at Scale</u>. PLoS Comput Biol. 2021. 17:e1009131. doi: https://doi.org/10.1371/journal.pcbi.1009131</ol>
           <ol> Chen, J., Madireddi, S., Nagarkar, D., <b>Migdał, M.</b>, et al. <u>In silico tools for accurate HLA and KIR inference from clinical sequencing data empower immunogenetics on individual-patient and population scales</u>. Briefings in Bioinformatics. 2021. doi: 10.1093/bib/bbaa223.</ol>
           <ol>Pawlak, M., Kedzierska, K.Z., <b>Migdal, M.</b>, Nahia, K.A., Ramilowski, J.A., Bugajski, L., Hashimoto, K., Marconi, A., Piwocka, K., Carninci, P., Winata, C.L. <u>Dynamics of cardiomyocyte transcriptome and chromatin landscape demarcates key events of heart development</u>. Genome Res.. 2019. 29:506-519. doi: https://doi.org/10.1101/gr.244491.118</ol>
         </ul>
       </section>
        </div>
      </div>
  </div>

  <footer class="footer">
    <div class="content has-text-left">
        <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">
            <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a><br />
            Background image is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License
        </a>.
    </div>
</body>

</html>
